src/tokens/TokenDetails.tsx

```diff
@@ -210,8 +209,7 @@ export const useActions = (token: TokenBalance) => {
       },
```

Comment:
I'll change this logic, if the user doesn't have any balance then I'll send them to the swap screen but with this token in the "to token"

Reply:
No, I will do this in another PR. Because the swap screen doesn't support a `toTokenId` as initial param. So this will need some changes there.
I think it's fine to merge this PR without this.